charts/ceph-csi-cephfs/README.md

```diff
@@ -26,23 +26,23 @@ To install the Chart into your Kubernetes cluster
 
 - For helm 2.x
 
-    ```bash
```

Comment:
It looks like there are more changes than expected in this file related to formatting, Do we need this change?

Reply:
I could revert the formatting changes. The md files inside charts are using the outdated syntax. Prettier auto formatted them and I decided to stick with it.

What would you suggest?

Reply:
i would suggest keeping the changes minimal and relevant to the PR as different developers might use different prettier configurations

Reply:
Done!